_data/reference.yml

```diff
@@ -25,7 +25,10 @@ toc:
     section:
     - docs/admin/accessing-the-api.md
     - docs/admin/authentication.md
-    - docs/admin/authorization.md
```

Comment:
@chenopis can you verify this is a good way to group a set of documents around authorization? the main topic is becoming huge, and I think we'll eventually want a page dedicated to each authz plugin. I *think* doing it this way preserves links to the old page (the old page is the index of the directory named identically to the old page)